lib/WorkflowHifi2Genome.groovy

```diff
@@ -0,0 +1,58 @@
+//
+// This file holds several functions specific to the multiqc configs for Arcadia-Science/hifi2genome workflow
+//
```

Comment:
where did this file come from?

Reply:
oooh did you have these ripped out and then you had to go back to the nf-core template to add the multiqc stuff?

Reply:
yes, it is exactly the same of what comes with the nf-core template but Mert ripped it out because I think he didn't like the extra multiqc things, so I had to put it back in